src/drivers/cookie.js

```diff
@@ -0,0 +1,38 @@
+var cookies = require('browser-cookies');
+
+var CookieStore = module.exports = function CookieStore () { };
+CookieStore.backendName = 'cookie';
+
+CookieStore.supported = function () {
+	var ret = false;
+	if (typeof document !== 'undefined') {
+		try {
+			// Create cookie
+			document.cookie = 'cookietest=1';
+			ret = document.cookie.indexOf('cookietest=') !== -1;
+			// Delete cookie
+			document.cookie = 'cookietest=; expires=Thu, 01-Jan-1970 00:00:01 GMT';
+		} catch (e) {
+			ret = false;
+		}
+	}
+	return ret;
+};
+
+CookieStore.prototype.getItem = cookies.get;
+CookieStore.prototype.removeItem = cookies.erase;
+
+CookieStore.prototype.setItem = function (key, val, params = {}) {
+	// Convert expires to days, because that is what the cookies module takes
+	if (typeof params.expires === 'number') {
+		params.expires = params.expires / (60 * 60 * 24);
+	}
+	return cookies.set(key, val, params);
+};
+
+CookieStore.prototype.clear = function () {
+	var c = cookies.get();
```

Comment:
Points to my fork of the repo, so you can run them now.  Once it all gets merged and published we can point back at the public package.